core/src/main/java/org/elasticsearch/discovery/DiscoveryStats.java

```diff
@@ -34,25 +36,42 @@
     @Nullable
     private final PendingClusterStateStats queueStats;
 
-    public DiscoveryStats(PendingClusterStateStats queueStats) {
+    @Nullable
```

Comment:
I do not think we need the `@Nullable` annotation on a private field (I question its usefulness in general).

Reply:
Should I remove the one above too?